src/lib/autoinstall/clients/clone_system.rb

```diff
@@ -151,8 +152,11 @@ def clone_system(options)
             # always clone general
             Yast::ProductControl.clone_modules + ["general"]
           end
+
         Yast::AutoinstClone.Process(target: target)
         begin
+          ::FileUtils.touch(filename)
+          ::FileUtils.chmod(0o600, filename)
```

Comment:
JFYI: It can be done as a single line:

```ruby
File.open(filename, "w", 0o600).close
```

But that does not change the permissions if the file already exists. I'm not sure if that would be acceptable in this case.

Reply:
Well, I think it is not acceptable in this case. But thanks for the hint!